src/vmm/src/rpc_interface.rs

```diff
@@ -279,7 +282,18 @@ impl<'a> PrebootApiController<'a> {
                 .build_net_device(netif_body)
                 .map(|_| VmmData::Empty)
                 .map_err(VmmActionError::NetworkConfig),
-            LoadSnapshot(_snapshot_load_cfg) => Ok(VmmData::NotFound),
```

Comment:
`VmmData::NotFound` is not used anymore, but I don't know if we should keep it around for future APIs that will temporarily lack an implementation, what do you think?